DCHP/download.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <!-- Bootstrap CSS -->
+        <link
+            rel="stylesheet"
+            href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+            integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z"
+            crossorigin="anonymous"
+        >
+        <link rel="stylesheet" href="../CSS/StyleSheet.css">
+        <script src="https://kit.fontawesome.com/f7a97ee3aa.js" crossorigin="anonymous"></script>
+        <!-- Website Favicon -->
+        <link rel="shortcut icon" type="image/png" href="../Images/Logo.png">
+        <!-- Logo Font -->
+        <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@800&display=swap" rel="stylesheet">
+        <title>DCHP: Download</title>
+    </head>
+    <body>
+        <!-- Navigation Bar Start -->
+        <nav class="navbar navbar-expand-sm">
+            <!-- Brand -->
+            <a class="navbar-brand devsoc-nav-logo" href="index.html">[}EVSOC_</a>
+            <!-- Toggler/collapsibe Button -->
+            <button
+                class="navbar-toggler"
+                type="button"
+                data-toggle="collapse"
+                data-target=".navbar-collapse"
+            >
+                <span class="navbar-toggler-icon">
+                    <i class="fa fa-bars" style="color: #fff; font-size: 28px"></i>
+                </span>
+            </button>
+            <div class="collapse navbar-collapse" id="collapsibleNavbar">
+                <ul class="navbar-nav ml-auto">
+                    <!-- Page -->
+                    <li class="nav-item">
+                        <a class="nav-link devsoc-nav-item" href="home.html">
+                            <i class="fa fa-fw fa-home" style="color: #fff"></i>
+                            Home
+                        </a>
+                    </li>
```

Comment:
DCHP nav element needs adding to navbar in `DHCP/download.html`
```suggestion
                    </li>
                    <li class="nav-item">
                        <a class="nav-link devsoc-nav-item" href="./introduction.html">
                            <i class="fa fa-fw fa-code" style="color:#fff"></i>
                            DCHP++
                        </a>
                    </li>
```